upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -919,42 +928,97 @@ function verify_pid_limit(){
     rm -f ${TEMP_MMHEALTH_FILE} >> /dev/null
 }
 
+function is_metrodr_setup(){
+    metrodrinstances=$(oc get metrodrs -n $FUSIONNS | tail -n +2 | wc -l)
+    if [ $metrodrinstances -gt 0 ]; then
+        echo 1
```

Comment:
please do not echo 1 or 0... use these silently. We want console output clean as much as possible

Reply:
remove echo from the method